doc/tutorials/rt_linux.rst

```diff
@@ -44,71 +44,75 @@ system on Intel KBL NUC with a SATA SSD as ``/dev/sda`` and an NVME SSD as
-      "NVME Support", delete ``# CONFIG_BLK_DEV_NVME is not set`` and add two lines under "NVME Support" to enable
-      the NVME driver in RT kernel::
-
-         CONFIG_NVME_CORE=y
-         CONFIG_BLK_DEV_NVME=y
-
-   #. Build the RT kernel::
-
-         $ cp kernel_config_uos .config
-         $ make targz-pkg
-
-      Choose "Fully Preemptible Kernel (RT)" when prompted, and
-      choose default for all the other options.
-
-   #. Copy the generated package to SOS::
-
-         $ scp linux-4.19.28-rt18-quilt-2e5dc0ac-dirty-x86.tar.gz <user name>@<SOS ip>:~/
-
-#. Configure the system on SOS
-
-
-   a. Extract kernel boot and lib modules from the package::
-
-         $ cd ~/
-         $ tar xzvf linux-4.19.28-rt18-quilt-2e5dc0ac-dirty-x86.tar.gz
-
-   #. Copy the extracted lib modules to NVME SSD::
-
-         $ mount /dev/nvme0n1p3 /mnt
-         $ cp -r ~/lib/modules/4.19.28-rt18-quilt-2e5dc0ac-dirty /mnt/lib/modules
-
-   #. Edit and run the ``launch_hard_rt_vm.sh`` script to launch the UOS.
-      A sample ``launch_hard_rt_vm.sh`` is included in the Clear Linux
-      release, and is also available in the acrn-hypervisor/devicemodel
-      GitHub repo (in the samples folder).
-
-      You'll need to modify two places:
-
-      1. Replace ``/root/rt_uos_kernel`` with ``~/boot/vmlinuz-4.19.28-rt18-quilt-2e5dc0ac-dirty``
-      #. Replace ``root=/dev/sda3`` with ``root=/dev/nvme0n1p3``
-
-   #. Run the launch script::
-
-         $ sudo ./launch_hard_rt_vm.sh
+#. Set up and launch RT-LaaG
+
+   a. Add kernel-lts2018-preempt-rt bundle from CL (root)::
+
+         # swupd bundle-add kernel-lts2018-preempt-rt
+
+   #. Copy preempt-rt module to NVMe disk::
+
+         # mount /dev/nvme0n1p3 /mnt
+         # ls /usr/lib/modules/
+         4.19.31-5.iot-lts2018-preempt-rt/ 
+         4.19.36-48.iot-lts2018/           
+         4.19.36-48.iot-lts2018-sos/       
+         5.0.10-7.preempt_rt/              
+         5.0.14-753.native/
+         # cp -r /usr/lib/modules/4.19.31-6.iot-lts2018-preempt-rt /mnt/lib/modules/
+         # cd ~ && umount /mnt && sync
+   
+   #. Get your NVMe pass-through IDs (mine is [01:00.0] and [8086:f1a6] )::
+         
+         # lspci -nn | grep SSD
+         01:00.0 Non-Volatile memory controller [0108]: Intel Corporation SSD Pro 7600p/760p/E 6100p Series [8086:f1a6] (rev 03)
+
+   #. Modify launch_hard_rt_vm.sh script::
+         
+         # vim /usr/share/acrn/samples/nuc/launch_hard_rt_vm.sh        
+      
+         <Modify the passthru_bdf with your NVMe pass-through IDs>
+         passthru_bdf=(
+         ["eth"]="0000:00:1f.6"
+         ["sata"]="0000:00:17.0"
+         )
+
+         TO:
+         passthru_bdf=(
+         ["eth"]="0000:00:1f.6"
+         ["sata"]="0000:01:00.0"
+         )
+      
+         <Modify boot kernel to RT>
+         -k /root/rt_uos_kernel \
+      
+         TO:
+         -k /usr/lib/kernel/default-iot-lts2018-preempt-rt \
```

Comment:
It would be better to remove that instruction from the tutorial and instead update the script we provide.

Reply:
”default-iot-lts2018-preempt-rt“ is a soft connection.It will point to the latest RT kernel.

Reply:
”default-iot-lts2018-preempt-rt“ is a soft connection.It will point to the latest RT kernel.

Reply:
'default-iot-lts2018-preempt-rt' is a soft connection . it will point at the latest rt kernel